### DISCUSSION TITLE
URL for pages contains wrong characters

### DISCUSSION THREAD

**Discussion by LBoese** [givanz/Vvveb#115] [General]
When a new page is created or an existing page is edited you can see in the admin page that the url contains an additional dot that is not necessary.

For example the incorrect URL is \\.test.com instead of \\test.com 

I download the latest version yesterday and installed it on a normal webserver.

**Comment by givanz** [comment]
Thanks for the bug report, It's fixed in the latest update https://github.com/givanz/Vvveb/commit/6f6a9fac6d6ac36498679c4adde48a73e307a311
